salt/files/cloud.providers.d/ec2.conf

```diff
@@ -2,13 +2,13 @@
 {% set cloud = salt['pillar.get']('salt:cloud', {}) -%}
 ec2_ubuntu_public:
   minion:
-    master: {{ cloud['master'] }}
+    master: {{ cloud.get('master', 'salt') }}
```

Comment:
I'm actually not sure this is even using defaults.yaml correctly. Or at least it's nothing like the way it's described in the documentation under [defaults.get](http://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.defaults.html). But yes, for purpose of this PR I was aiming for the minimum changes to the default templates to make them not-fail when recurse included files the user isn't using.